Stage rl/standard debug sockets (#876)
Shopify/vscode-ruby-lsp: README.md

Available commands are listed below and can always be found by searching for the `Ruby LSP` prefix in the command
palette (Default hotkey: CMD + SHIFT + P).

| Command                              | Description                                            |
| -------------------------------------- | -------------------------------------------------------- |
| Ruby LSP: Start                      | Start the Ruby LSP server                              |
| Ruby LSP: Restart                    | Restart the Ruby LSP server                            |
| Ruby LSP: Stop                       | Stop the Ruby LSP server                               |

> | Ruby LSP: Update language server gem | Updates the`ruby-lsp` server gem to the latest version |

Reviewer: What changed here? Can we remove this change?

Author: Sure, no change intended, vscode Markdown editor got too aggresive

}
```

> ### Debugging rails applications

Reviewer: ```suggestion
### Debugging live processes
```

Author: For people searching for Rails debugging I suggest keeping at least `Debugging live processes (e.g. Rails applications)`

Reviewer: Agreed. Can we switch the title and instead add a phrase explaining what this is then? Something along the lines of

`Instead of launching a process to debug every time, you may want to attach the VS Code debugger to an existing process, such as a Rails server. Follow these instructions to do so.`

Install `debug` gem. Verify by running `bundle exec rdbg -v`

> Make sure you're running your application with rdbg, e.g. `bundle exec rdbg -O -n -c -- bin/rails server -p 3000`

Reviewer: ```suggestion
Run your application with the debugger attached, so that the extension can connect to it.

bundle exec rdbg -O -n -c -- bin/rails server -p 3000
```

> You should be able to use vscode debugger attach configuration to connect to your running app

Reviewer: Launching the attach task should display a list of existing sockets for your running applications. Select the one you'd like to connect to and start debugging.

> For better integrated rails tests support also install `ruby-lsp-rails` gem

Reviewer: ```suggestion
For better integrated rails tests support also install `ruby-lsp-rails` gem.
```
